include/maliput_sparse/geometry/utility/geometry.h

```diff
@@ -46,6 +54,34 @@ namespace utility {
 /// @returns The centerline.
 LineString3d ComputeCenterline3d(const LineString3d& left, const LineString3d& right);
 
+/// Returns the piecewise linearly interpolated point at the given distance.
+/// @param line_string the line_string to iterate. Size must be >0.
+/// @param p distance along line_string.
+/// @return The interpolated point (a new point if not perfectly matching).
+maliput::math::Vector3 InterpolatedPointAtP(const LineString3d& line_string, double p);
+
+/// Returns the slope of a @p line_string for a given @p p .
+/// The result is expected to be contained within [-1., 1.].
```

Comment:
It is not clear to me why the result is constrained between -1 and 1. Can you expand?

Reply:
Sorry, it was an old comment. It is between -inf and inf now.

Reply:
Fixed.